HST/tests/test_organize_files.py

```
@@ -100,3 +100,22 @@ def test_clean_up_staging_dir_only_other(mock_rmtree, mock_listdir, mock_isdir,
     organize_files.clean_up_staging_dir(proposal_id, fake_logger)
     # No rmtree calls for dirs not matching prefix or mastDownload
     assert mock_rmtree.call_count == 0
+
+@mock.patch('HST.organize_files.HST_DIR', {'staging': '/fake/staging'})
+@mock.patch('os.path.isdir')
+@mock.patch('os.listdir')
+@mock.patch('shutil.rmtree')
+def test_clean_up_staging_dir_all_programs(mock_rmtree, mock_listdir, mock_isdir, fake_logger, patch_helpers):
```

Ruff (0.15.14)

[warning] Unused function argument: `patch_helpers`

(ARG001)